website/_containers/begin/2013-10-30-setup-your-machine.md

```diff
@@ -134,7 +134,7 @@ To test if you have either GCC or clang, type `$ gcc` or `$ clang` into your ter
 
 ### pip
 
-[pip][9], stands for “python install python”, is a tool for installing and managing Python packages. Within your Terminal application, use the following commands (ignore the leading `$` as that is your terminal prompt) for downloading & installing. It may prompt you for your computer login password.
+[pip][9] is a tool for installing Python packages. Within your Terminal application, use the following commands (ignore the leading `$` as that is your terminal prompt) for downloading & installing. It may prompt you for your computer login password.
```

Comment:
I like this!  I know it is I who proposed “pip is a tool for installing Python packages”, but seeing this I think now that it’s good to keep “installing and managing” (because we can update, uninstall, query, etc.).  Would you mind updating that?